tasks/fix-cat2.yml

```diff
@@ -1285,118 +1329,210 @@
       - medium
       - patch
       - RHEL-07-040250
+      - firewalld
 
 - name: "MEDIUM | RHEL-07-040260 | PATCH | All networked systems must have SSH installed."
-  command: "true"
+  yum:
+      name:
+          - openssh-clients
+          - openssh-server
+      state: latest
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-040260
+      - ssh
 
 - name: "MEDIUM | RHEL-07-040261 | PATCH | All networked systems must use SSH for confidentiality and integrity of transmitted and received information as well as information during preparation for transmission."
-  command: "true"
+  service:
+      name: sshd
+      state: started
+      enabled: yes
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-040261
+      - ssh
 
-- name: "MEDIUM | RHEL-07-040290 | PATCH | The operating system must enable an application firewall, if available."
-  command: "true"
+- name: "MEDIUM | RHEL-07-040290 | PATCH | The system must use a local firewall."
```

Comment:
Nothing wrong with this task, but I'm wondering if there are previous findings that manipulate the firewall that may fail if the firewall is not yet installed. Something to check.